### ISSUE TITLE
include WGS database

### ISSUE THREAD

**Issue by MRuinels** [infphilo/centrifuge#110]
Hello I am new using Centrifuge. I am downloading now the databases and indices for complete genomes and nt/nr NCBI database. I was just wondering if it would be possible to include also a WGS database. Thanks :)

**Comment by khyox** [comment]
Yes, it is, you can include WGS database(s) when creating your Centrifuge database.

**Comment by khyox** [comment]
Yes, WGS is a huge repository and, as far as I know, you have to download the databases on a project basis approach. The good news is that the taxonomy is the same.
